src/components/StakingPage/index.tsx

```diff
@@ -15,13 +15,27 @@ import { useTokenApprove, ApprovalState } from '../../hooks/useTokenApprove';
 import { useHakkaStake, StakeState } from '../../hooks/staking/useHakkaStake';
 import StakePositionItem from './StakePositionItem/index';
 import {
-  ChainId, HAKKA, STAKING_ADDRESSES, stakingMonth,
+  ChainId,
+  HAKKA,
+  STAKING_ADDRESSES,
+  stakingMonth,
 } from '../../constants';
 import { tryParseAmount } from '../../utils';
 import { useWalletModalToggle } from '../../state/application/hooks';
 import withConnectWalletCheckWrapper from '../../hoc/withConnectWalletCheckWrapper';
 import withApproveTokenCheckWrapper from '../../hoc/withApproveTokenCheckWrapper';
 import withWrongNetworkCheckWrapper from '../../hoc/withWrongNetworkCheckWrapper';
+import { TabGroup } from '../Common/TabGroup';
+
+import EthIcon from '../../images/icons/iconEthereum.svg';
+import SVGIcon from '../../images/icons/iconBSC.svg';
+import PolygonIcon from '../../images/icons/iconPolygon.svg';
+
+const TabsMocking = [
```

Comment:
Why this is 'Mocking'? I think it's a config for Tabs, just call it config could be fine

Reply:
Will be modified when this integrate with correct action hooks